temporal/api/cloud/cloudservice/v1/request_response.proto

```diff
@@ -107,3 +109,99 @@ message GetAsyncOperationResponse {
     // The async operation
     temporal.api.cloud.operation.v1.AsyncOperation async_operation = 1;
 }
+
+message CreateNamespaceRequest {
+    // The namespace specification.
+    temporal.api.cloud.namespace.v1.NamespaceSpec spec = 2;
+    // The id to use for this async operation.
+    // Optional, if not provided a random id will be generated.
+    string async_operation_id = 3;
+}
+
+message CreateNamespaceResponse {
+    // The namespace that was created.
+    string namespace = 1;
+    // The async operation.
+    temporal.api.cloud.operation.v1.AsyncOperation async_operation = 2;
+}
+
+message GetNamespacesRequest {
+    // The requested size of the page to retrieve.
+    // Cannot exceed 1000. 
+    // Optional, defaults to 100. 
+    int32 page_size = 1;
+    // The page token if this is continuing from another response.
+    // Optional, defaults to empty.
+    string page_token = 2;
```

Comment:
Will we add more filters in a follow-up? Things like region (or regions) etc.

Reply:
Absolutely with time.